src/3d/Viewport3d.tsx

```diff
@@ -152,14 +154,17 @@ const Viewport3d = () => {
                     shadow-camera-top={10}
                     shadow-camera-bottom={-10} intensity={0.8} />
 
-                <Grid/>
+                <Grid />
                 <Plane receiveShadow={true} scale={100} rotation-x={-Math.PI * 0.5} position-y={-0.5}>
                     <shadowMaterial opacity={0.1} />
                 </Plane>
             </Canvas>
             <Toolbar onZoomExtents={zoomExtents} onTogglePerspective={handleTogglePerspective} onNewFile={handleNewFile}></Toolbar>
-            <HelpAndSoundButtons onShowHelp={() => setShowHelp(!showHelp)}/>
-            {showHelp && <HelpModal onClose={() => setShowHelp(false)}/>}
+            <HelpAndSoundButtons onShowHelp={() => setShowHelp(!showHelp)} />
+            {showHelp && <HelpModal onClose={() => setShowHelp(false)} />}
```

Comment:
move 163 and 164 after the mobile buttons to that the help modal displays on top :)